### PR TITLE
Use ',' as a function argument separator in the docs

### LINKED ISSUE #999: Always use `,` as a function parameters separator in the docs
### Description

Although the separator for function parameters can be [configured](https://handsontable.github.io/hyperformula/api/interfaces/configparams.html#functionargseparator), `,` is the default separator and it should be used consistently in our docs.

We use `;` instead of ',' in:
- https://handsontable.github.io/hyperformula/guide/built-in-functions.html#math-and-trigonometry
- https://handsontable.com/blog/articles/2021/8/8-examples-of-useful-excel-functions-in-hyperformula
- ...?

### Links

https://handsoncode.slack.com/archives/C031GBX62SE/p1653572684792079 (internal discussion)

### PR DESCRIPTION
**Pull request by sequba**
### How did you test your changes?

Local docs build

### Types of changes

- [ ] Breaking change (a fix or a feature because of which an existing functionality doesn't work as expected anymore)
- [ ] New feature or improvement (a non-breaking change that adds functionality)
- [ ] Bug fix (a non-breaking change that fixes an issue)
- [ ] Additional language file, or a change to an existing language file (translations)
- [x] Change to the documentation

### Related issues:
Fixes #999

### Checklist:

- [x] My change is compliant with the [OpenDocument](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html) standard.
- [x] My change is compatible with Microsoft Excel.
- [x] My change is compatible with Google Sheets.
- [x] My code follows the code style of this project.
- [ ] I described my changes in the [CHANGELOG.md](https://github.com/handsontable/hyperformula/blob/master/CHANGELOG.md) file.
- [ ] My changes require a documentation update.
- [ ] My changes require a migration guide.

### CODE REVIEW

File: docs/guide/known-limitations.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ returns a cycle error regardless of whether TRUE or FALSE causes
 a circular reference.
 * There is no data validation against named ranges. For example,
 you can't compare the arguments in a formula like this:
-=IF(firstRange>secondRange; TRUE; FALSE).
```
Comment (warpech) [inline docs/guide/known-limitations.md:19]: Is `secondRange` omitted on purpose?

**Review by warpech — CHANGES_REQUESTED** [review]
Great job! I verified that I cannot find any other `;` that should be changed to `,`. I verified that Markdown seems correct after the changes.

I have just one question, see below

File: docs/guide/known-limitations.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ returns a cycle error regardless of whether TRUE or FALSE causes
...
+=IF(firstRange>, TRUE, FALSE).
```
Comment (warpech) [inline docs/guide/known-limitations.md:19]: Is `firstRange>secondRange` changed to `firstRange>` on purpose?

**Review by warpech — CHANGES_REQUESTED** [review]

File: docs/guide/known-limitations.md (hunk 2 above)
Comment (sequba) [reply docs/guide/known-limitations.md]: It was a mistake. Thanks for spotting it. It's fixed.

**Review by warpech — APPROVED** [review]
Looking good!

**Pull request merged**